bedrock/firefox/tests/test_base.py

```diff
@@ -756,17 +756,17 @@ def setUp(self):
     def test_fx_firstrun_40_0(self, render_mock):
         """Should use default firstrun template"""
         req = self.rf.get("/en-US/firefox/firstrun/")
-        self.view(req, version="40.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/firstrun/firstrun.html"]
+        resp = self.view(req, version="40.0")
+        assert resp.status_code == 301
+        assert resp["location"].endswith("/firefox/new/?reason=outdated")
 
     @override_settings(DEV=True)
     def test_fx_firstrun_56_0(self, render_mock):
         """Should use the default firstrun template"""
         req = self.rf.get("/en-US/firefox/firstrun/")
-        self.view(req, version="56.0a2")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/firstrun/firstrun.html"]
+        resp = self.view(req, version="56.0a2")
+        assert resp.status_code == 301
+        assert resp["location"].endswith("/firefox/developer/")
 
     @override_settings(DEV=True)
```

Comment:
Question: Could we now delete this test, since it's essentially the same as `test_fx_firstrun_release_channel`?